src/Waypoint/HomeGlue.cpp

```diff
@@ -27,6 +27,13 @@ FindHomeId(Waypoints &waypoints,
 
   settings.home_location = wp->location;
   settings.home_location_available = true;
+
+  if (wp->has_elevation) {
```

Comment:
You are tracking the state in separate variables. Please check home_elevation for value or nullptr instead.

Reply:
Thanks for reviewing and commenting. I copied this approach from Top Hat, and it's like how elevation is managed in the Waypoint structure (in XCSoar). This structure has "has_elevation" and "elevation" members, and the latter is to be used only if the former is true. "home_elevation" isn't a pointer, so it can never be nullptr, right? I suppose we could choose some impossible value (e.g., -999m) to which to set home_elevation to signify that it's an invalid/unavailable value (thus negating the need for the separate home_elevation_available variable), but this is inconsistent with how it's done in the Waypoint structure. Would you like me to do something like that... or leave it as is? Thanks.